### DISCUSSION TITLE
Proposal: Drop dependency on pytz in favor of zoneinfo

### DISCUSSION THREAD

**Discussion by aDramaQueen** [encode/django-rest-framework#8562] [Ideas & Suggestions] [answered]
Hi,
pytz is/was a good llibrary. But it's obsolete since [PEP 615](https://peps.python.org/pep-0615/). In the long run, you should not use this 3rd party package, but the language integrated functionality.
Even big frameworks like Django [(Django 4.0 Release Info)](https://docs.djangoproject.com/en/4.0/releases/4.0/#what-s-new-in-django-4-0) already dropped pytz.

**Comment by pjrobertson** [comment, marked as answer]
Agreed. Django now uses `zoneinfo` by default, and backports `backports.zoneinfo` if the user is not running pyhton 3.9.

pytz support will be dropped in Django 5.0. Django-rest should switch to using `zoneinfo` by default.

More info from [here](https://docs.djangoproject.com/en/4.0/topics/i18n/timezones/#overview)

> Time zone support uses [zoneinfo](https://docs.python.org/3/library/zoneinfo.html#module-zoneinfo), which is part of the Python standard library from Python 3.9. The backports.zoneinfo package is automatically installed alongside Django if you are using Python 3.8.
> Changed in Django 3.2:

> [zoneinfo](https://docs.python.org/3/library/zoneinfo.html#module-zoneinfo) was made the default timezone implementation. You may continue to use [pytz](http://pytz.sourceforge.net/) during the 4.x release cycle via the [USE_DEPRECATED_PYTZ](https://docs.djangoproject.com/en/4.0/ref/settings/#std-setting-USE_DEPRECATED_PYTZ) setting.
